api/v1beta1/ccruntime_types.go

```diff
@@ -313,6 +313,14 @@ type FailedNodeStatus struct {
 	Error string `json:"error"`
 }
 
+// RuntimeClass holds the name and the snapshotter to be used by a runtime class
+type RuntimeClass struct {
+	// Name of the runtime class
+	Name string `json:"name"`
+	// The snapshotter to be used by the runtime class
+	Snapshotter string `json:"snapshotter"`
```

Comment:
Any other inputs needed for the snapshotter ?

Reply:
To be more clear, this is the counter part change on the Kata Containers side: https://github.com/kata-containers/kata-containers/pull/7715/files